.evergreen/config_generator/components/make_docs.py

```diff
@@ -0,0 +1,140 @@
+from shrub.v3.evg_command import EvgCommandType
+from shrub.v3.evg_command import s3_put
+from shrub.v3.evg_task import EvgTask
+
+from config_generator.etc.function import Function
+from config_generator.etc.function import merge_defns
+from config_generator.etc.utils import bash_exec
+
+
+class MakeDocs(Function):
+    name = "make-docs"
+    commands = [
+        bash_exec(
+            command_type=EvgCommandType.TEST,
+            working_dir="mongoc",
+            include_expansions_in_env=["distro_id"],
+            script="""\
+                set -o errexit
+                bash tools/poetry.sh install --with=docs
+                bash tools/poetry.sh run \
+                    bash .evergreen/scripts/build-docs.sh
```

Comment:
```suggestion
                bash tools/poetry.sh run bash .evergreen/scripts/build-docs.sh
```
Avoids messy formatting in the generated config file.